src/test/java/DonkeyTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file contains sample JUnit test cases for Horse.java
+ * You should NOT modify this file!
+ */
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+
+public class DonkeyTest {
+    DonkeySupercar h;
+
+    @Before
+    public void setUp() throws Exception {
+        h = new DonkeySupercar();
+    }
+
+    @Test(timeout = 50)
+    public void TestSound() {
+        assertEquals("Neigh!", h.sound());
```

Comment:
Poor test case since DonkeySupercar doesn't make the "Neigh!" sound but "Vrooom!" instead.